content/04.workflows-and-software.md

```diff
@@ -8,7 +8,7 @@ Properly managing and executing all of these steps is vital, but can be both tim
 
 The emergence and maturation of workflow systems designed with bioinformatic challenges in mind has revolutionized computing in data intensive biology [@doi:10.1038/s41592-018-0046-7].
 Workflow systems contain powerful infrastructure for workflow management that can coordinate runtime behavior, self-monitor progress and resource usage, and compile reports documenting the results of a workflow (**Figure @fig:workflow**).
-These features ensure that the steps for data analysis are minimally documented and repeatable from start to finish.
+These features ensure that the steps for data analysis are at least minimally documented and repeatable from start to finish.
```

Comment:
"at least minimally documented and repeatable" - just to clarify, this is "completely repeatable" and "at least minimally documented"?